openid-4-verifiable-presentations-1_0.md

```diff
@@ -1464,6 +1464,132 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
+
+### Client ID Scheme web-origin
+
+This profile defines the new Client ID Scheme `web-origin`. If this scheme is used, the Client Identifier is the origin of the Verifier web site as asserted by the browser towards the Wallet. 
+
+## Response
+
+Every OID4VP request MUST result in a response being provided through the W3C Digital Credentials API. In all cases, except if the response mode `direct_post` is used, the response MUST be follow the rules specified for OID4VP Authorization Responses. If the response model `direct_post` is used, the Wallet will return the URL the Direct Post Endpoint of the Verifier has returned. 
+
+The following is an example of an OID4VP Authorization Response through the API: 
+
+```json
+{
+  "presentation_submission": "...",
+  "vp_token": "..."
+}
+```
+
+Note: All mechanisms for cryptographically protecting the OID4VP response MAY be utilized with the W3C Digital Credentials API, too. 
+
+## Example Flows
+
+As OID4VP is pretty flexible, this section shall shed some light on what flows the authors deem especially useful.  
+
+### Default Flow
+
+In the Default flow, the Verifier sends all the OID4VP request data in the `request` parameter and reveices the result in the `result` parameter. In this case, the Wallet will use the Verifier origin as asserted by the Browser as the Verifers's Client Identifier.  
+
+### Request URI Flow
+
+In this flow, the Verifier sends a `request` with enough data to select the Wallet but the full OID4VP Authorization Request is fetched by the Wallet from an endpoint hosted by the Verifier. This allows the Verifier to sign and optionally encrypt the request data. 
+
+The Verifier should use the `request_uri` parameter in conjunction with the `request_uri_method` parameter with a value of `post`. This will cause the Wallet to send a HTTPS POST request to the endpoint determined by `request_uri`. The request should also contain the `client_metadata` parameter with the Verifier's capabilities in order to allow the Wallet to only offer the capabilities common between both parties to the Verifier. 
+
+This is a example of such a request:
+
+```JavaScript
+const credential = await navigator.identity.get({
+  digital: {
+    providers: [{
+      protocol: "urn:openid.net:oid4vp",
+      request: JSON.stringify({
+        "client_id":"https//client.example.org",
+        "client_id_scheme":"entity_id",
+        "presentation_definition":"...",
+        "request_uri":"https://client.example.org/request",
+        "request_uri_method":"post"})
```

Comment:
```suggestion
        "request_uri_method":"post"}
```
corresponding to the Suggested change above https://github.com/openid/OpenID4VP/pull/155/files#r1567773088

Reply:
reworked the profile to use signed requests